packages/vscode/src/index.ts

```diff
@@ -91,6 +104,50 @@ export async function activate(context: ExtensionContext) {
 	context.subscriptions.push(
 		commands.registerCommand('astro.restartLanguageServer', async () => {
 			await restartClient(true);
+		}),
+		commands.registerCommand('astro.openDevServerPanel', async () => {
+			// Let's first try to find if we have a dev server currently running already
+			let url = await getDevServerUrl();
+
+			// If not, let's run one
+			if (!url) {
+				const terminal = window.createTerminal({ name: 'astro:dev' });
+				terminal.show();
+				terminal.sendText('node ./node_modules/astro/astro.js dev');
```

Comment:
Is the intention to have users already include Astro in their package.json files

Reply:
Yes, this assumes that Astro and its requisites are installed and ready to be started

For users who don't have Astro installed, the language server send a notification on startup about how the experience might be degraded due to that, so it doesn't bother me too much to assume that users have Astro ready to go (this is also inline with how other language servers tend to work)